packages/greenbox-data/data/iotms.ts

```diff
@@ -339,6 +339,7 @@ const iotms: IotMDef[] = [
   { id: 11099, month: 1, year: 2023, type: "campground", item: "packet of rock seeds" }, // rock garden guide
   { id: 11115, month: 2, year: 2023, type: "item", item: "S.I.T. Course Completion Certificate" }, // S.I.T. Course Voucher
   { id: 11168, month: 3, year: 2023, type: "item", item: "closed-circuit pay phone" }, // closed-circuit phone system
+  { id: 11186, month: 4, year: 2023, type: "item", item: "cursed monkey's paw" }, // cursed monkey glove
```

Comment:
I think the id should be 11187 instead, so we get the packaged version, right?

I tried doing this myself but it was showing as seal-clubbing club even though the data for the glove is showing up in the mafia data file. Didn't get time to troubleshoot through exactly what was breaking with it so let me know if you all have seen this issue before.

Reply:
this is correct